lib/srv/app/transport.go

```diff
@@ -111,6 +112,26 @@ func newTransport(ctx context.Context, c *transportConfig) (*transport, error) {
 // RoundTrip will rewrite the request, forward the request to the target
 // application, emit an event to the audit log, then rewrite the response.
 func (t *transport) RoundTrip(r *http.Request) (*http.Response, error) {
+	// Check if the request path needs re-writing. This occurs when the URI
+	// contains a path like http://localhost:8080/app/acme, but the request comes
+	// to https://publicAddr. In that case do a 302 to the correct path instead
+	// of doing path re-writing on all requests. This is a workaround to make
```

Comment:
Will this mean my Kubernetes dashboard will work out of the box, redirecting directly to the URI of the dashboard?

Reply:
Yes, cutting a release right now, will be in 5.0.0-beta.9.